snarkos/updater.rs

```diff
@@ -34,7 +35,7 @@ impl Updater {
 
         let mut output = "List of available versions\n".to_string();
         for release in releases {
-            output += &format!("  * {}\n", release.version);
+            let _ = writeln!(output, "  * {}", release.version);
```

Comment:
Does this need the `let _ = ` part?

Reply:
there could be a `?` at the end instead, but since this macro is writing to memory (`output`), there is no need to account for any errors

Reply:
And the error itself isn't castable to the specific type returned from the function, at least not currently.

Reply:
ah cool, I missed the fact that the error was incompatible :+1: